packages/dev/core/src/Materials/GreasedLine/greasedLinePluginMaterial.ts

```diff
@@ -321,164 +354,21 @@ export class GreasedLinePluginMaterial extends MaterialPluginBase implements IGr
-                    float grlBaseWidth = grl_aspect_resolution_lineWidth.w;
-
-
-                    vec3 grlPrevious = grl_previousAndSide.xyz;
-                    float grlSide = grl_previousAndSide.w;
-
-                    vec3 grlNext = grl_nextAndCounters.xyz;
-                    grlCounters = grl_nextAndCounters.w;
-
-                    mat4 grlMatrix = viewProjection * finalWorld;
-                    vec4 grlFinalPosition = grlMatrix * vec4( positionUpdated , 1.0 );
-                    vec4 grlPrevPos = grlMatrix * vec4( grlPrevious + grlPositionOffset, 1.0 );
-                    vec4 grlNextPos = grlMatrix * vec4( grlNext + grlPositionOffset, 1.0 );
-
-                    vec2 grlCurrentP = grlFix( grlFinalPosition, grlAspect );
-                    vec2 grlPrevP = grlFix( grlPrevPos, grlAspect );
-                    vec2 grlNextP = grlFix( grlNextPos, grlAspect );
-
-                    float grlWidth = grlBaseWidth * grl_widths;
-
-                    vec2 grlDir;
-                    if( grlNextP == grlCurrentP ) grlDir = normalize( grlCurrentP - grlPrevP );
-                    else if( grlPrevP == grlCurrentP ) grlDir = normalize( grlNextP - grlCurrentP );
-                    else {
-                        vec2 grlDir1 = normalize( grlCurrentP - grlPrevP );
-                        vec2 grlDir2 = normalize( grlNextP - grlCurrentP );
-                        grlDir = normalize( grlDir1 + grlDir2 );
-                    }
-                    vec4 grlNormal = vec4( -grlDir.y, grlDir.x, 0., 1. );
-                    #ifdef GREASED_LINE_RIGHT_HANDED_COORDINATE_SYSTEM
-                        grlNormal.xy *= -.5 * grlWidth;
-                    #else
-                        grlNormal.xy *= .5 * grlWidth;
-                    #endif
-
-                    grlNormal *= grl_projection;
-
-                    #ifdef GREASED_LINE_SIZE_ATTENUATION
-                        grlNormal.xy *= grlFinalPosition.w;
-                        grlNormal.xy /= ( vec4( grl_aspect_resolution_lineWidth.yz, 0., 1. ) * grl_projection ).xy;
-                    #endif
-
-                    grlFinalPosition.xy += grlNormal.xy * grlSide;
-                    gl_Position = grlFinalPosition;
-
-                    vPositionW = vec3(grlFinalPosition);
-                #else
-                    grlCounters = grl_counters;
-                #endif
-                `,
-            };
-            this._cameraFacing && (obj["!gl_Position\\=viewProjection\\*worldPos;"] = "//"); // not needed for camera facing GRL
-            return obj;
+    override getCustomCode(shaderType: string, shaderLanguage = ShaderLanguage.GLSL): Nullable<{ [pointName: string]: string }> {
+        // Logger.Log("getCustomCode" + this._customShaderCodeGetter);
+        // if (!this._customShaderCodeGetter) {
+        //     return null;
+        // }
+        // return this._customShaderCodeGetter(shaderType, this._cameraFacing);

```

Comment:
```suggestion
```

Reply:
I removed this earlier already and seems it got back when I was undoing some changes... :-( Removed.